app/src/components/Category.js

```diff
@@ -0,0 +1,30 @@
+import styled, { ThemeProvider } from "styled-components";
+import theme from "../styles/theme.js";
+import CardList from "./CardList.js";
+
+const Header = styled.h3`
+  font-size: ${({ theme }) => theme.fontSize.medium};
+`;
+
+const Category = ({ dish, category }) => {
+  const title = {
+    side: "식탁을 풍성하게 하는 정갈한 밑반찬",
+    soup: "정성이 담긴 뜨끈뜨끈 국물 요리",
+    main: "모두가 좋아하는 든든한 메인 요리",
+  };
```

Comment:
상수로 정의해둔 점 아주 좋습니다!
상수인 것을 더 명시적으로 알 수 있도록 대문자를 활용해보면 더 좋겠습니다.
그리고, 이 title은 컴포넌트가 호출될 때마다 정의될 필요가 없으므로 컴포넌트 바깥으로 빼도 될 것 같습니다!

```suggestion
  const TITLE = {
    SIDE: "식탁을 풍성하게 하는 정갈한 밑반찬",
    SOUP: "정성이 담긴 뜨끈뜨끈 국물 요리",
    MAIN: "모두가 좋아하는 든든한 메인 요리",
  };
```